src/Qowaiv.Bson.MongoDB/QowaivBsonConverter.cs

```diff
@@ -1,24 +1,97 @@
 ﻿using MongoDB.Bson.Serialization;
 using System;
+using System.Linq;
+using System.Reflection;
 
 namespace Qowaiv.Bson.MongoDB
 {
+    /// <summary>Static helper class to register <see cref="QowaivBsonConverter{TSvo}"/> for specific types.</summary>
     public static class QowaivBsonConverter
     {
+        /// <summary>Registers all supported types of the assembly.</summary>
+        /// <param name="assemblyType">
+        /// A type that is part of the assembly to register a <see cref="QowaivBsonConverter{TSvo}"/> for.
+        /// </param>
+        public static void RegisterAssembly(Type assemblyType)
```

Comment:
I think it is quite common to use a type, instead of an assembly, as in 99.9% of the cases it wilt turn out you do somehting like `typeof(SomeType).Assembly` but you do not like it?! Do you like to see an extra overload, or just the `Assembly` argument variation?